lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -105,7 +105,8 @@ def cop_config_params(default_cfg, cfg)
       end
 
       def output_cop_param_comments(params, default_cfg)
-        output.puts "# Configuration parameters: #{params.join(', ')}."
+        config_params = params.reject { |p| p.start_with?('Supported') }
```

Comment:
I wonder, why use `start_with?` to get a partial match when you can do an exact match like this:
```rb
config_params = params - ['SupportedStyles']
```

Reply:
RuboCop has configurations that start with `Supported` except `SupportedStyles`. For example: `SupportedHashRocketStyles`

https://github.com/bbatsov/rubocop/blob/6fb5a84976567549302881de4923e21819040756/config/default.yml#L146

Reply:
Ah